verilog/tools/ls/lsp-parse-buffer.h

```diff
@@ -104,7 +109,7 @@ class BufferTrackerContainer {
   // Remove the buffer tracker for the given "uri".
   void Remove(const std::string &uri) { buffers_.erase(uri); }
 
-  ChangeCallback change_listener_ = nullptr;
+  std::unordered_map<std::string, ChangeCallback> change_listeners_;
```

Comment:
If we don't need the delete functionality of the change listener (see comment above) would a simple `std::vector<ChangeCallback>` be sufficient ?